imagetest/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+  - hopkiw
```

Comment:
I strongly disagree with a single approver. This is a resource owned by the team.

Reply:
this is just my first thought of how to ensure I approve all PRs in this specific subdirectory. If I were unavailable, you'd still be able to remove this OWNERS file if needed to bypass.

Do you have an alternative suggestion on how to accomplish this?

Reply:
I wasn't sure how I felt about this, but now that I've thought about it, I don't think this is necessary. While I understand wanting to review all of the changes in this subdirectory, the downsides of a not-quite-fully-reviewed PR are not as high as with some of the other code that we own. If a PR gets merged in and there were changes that would have been useful to make the code clearer or more correct, we can just send that in another PR. We don't need to create a whole new release if there is some minor change that comes in a separate PR.

It would perhaps be useful to have additional automated checks in this repo/subdir, but that's a separate discussion.